app/controllers/checksum_records_controller.rb

```diff
@@ -2,20 +2,6 @@
 class ChecksumRecordsController < ApplicationController
   before_action :set_checksum_record, only: [:show, :update, :destroy]
```

Comment:
doubt it matters, but this before_action line #3 in controller is referring to update and destroy methods

Reply:
i would be inclined to get rid of the `:update` and `:destroy` since the controller methods are gone, just to keep everything clear and tidy.

Reply:
whoa - where did that come from?